src/engineering/toast.mdx

```diff
@@ -0,0 +1,75 @@
+# Toast
+
+#### 5th March 2024 by Sean Quiambao
+
+## Why React Hot Toast?
```

Comment:
talk about why we needed these notifications? what was the main problem we ran into? and how did react hot toast fix this issue for us